hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/repair/om/FSORepairCLI.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.repair.om;
+
+import org.apache.hadoop.hdds.cli.SubcommandWithParent;
+import org.apache.hadoop.ozone.repair.OzoneRepair;
+import org.kohsuke.MetaInfServices;
+import picocli.CommandLine;
+
+import java.util.concurrent.Callable;
+
+/**
+ * Parser for scm.db file.
+ */
+@CommandLine.Command(
+    name = "fso-tree-repair",
+    description = "Identify and repair a disconnected FSO tree, and mark " +
+        "unreachable entries for deletion. OM should be " +
+        "stopped while this tool is run."
+)
+@MetaInfServices(SubcommandWithParent.class)
+public class FSORepairCLI implements Callable<Void>, SubcommandWithParent {
+
+  @CommandLine.ParentCommand
+  private OzoneRepair parent;
+
+  @CommandLine.Option(names = {"--db"},
+      required = true,
+      description = "Path to OM RocksDB")
+  private String dbPath;
+
+  @CommandLine.Option(names = {"--dry-run"},
```

Comment:
dry-run must be implicit option, and repair must be explicit, as person can miss dry-run usually,

Reply:
I think "implicit option" means it should be removed.  If I understand correctly, there are two modes: repair or not repair (a.k.a. dry run).  So if we have option `--repair`, then `--dry-run` is not needed.